sunpy/coordinates/tests/test_sun.py

```diff
@@ -467,7 +469,7 @@ def test_carrington_rotation_starttime(crot, julian_days):
 def test_carrington_rotation_roundtrip():
     t = Time('2010-1-1')
     crot = sun.carrington_rotation_number(t)
-    t_roundtrip = sun.carrington_rotation_time(crot)
+    t_roundtrip = sun.carrington_rotation_time(crot.value)
```

Comment:
```suggestion
    t_roundtrip = sun.carrington_rotation_time(crot)
```
Oops!  I hadn't intended to change the return type of `sunpy.coordinates.sun.carrington_rotation_number()`.  Can you revert this change here and change the return statement in `sunpy.coordinates.sun.carrington_rotation_number()` to `return actual.to_value(u.one)`?

Reply:
Fixed.